### PR TITLE
Added papers and modernised website

### PR DESCRIPTION
**Pull request by lordqwerty**
I've added ESORICS and STM 2017 papers to the list. 
Additionally I have:
* Screenshots updated to show TAMARIN 1.3.0
* Images centred
* CSS moved externally
* jQuery and BootStrap now use dedicated CDNs (improves loading time)
* Formatted HTML 

Screenshot of running website:
screenshot from 2017-10-16 21-53-38

### CODE REVIEW

**Comment by jdreier** [comment]
Thanks for the work!

I am wondering whether it would make sense to split the section on research papers in two: papers about Tamarin (new features, theory, etc.), and papers that use Tamarin (there we could add quite a few more from other authors).

**Comment by lordqwerty** [comment]
If you list the papers that should be in each section I'm happy to add the
commit.

Kind Regards,

Jorden Whitefield

**Comment by rsasse** [comment]
Cool, thanks a lot for your change here, Jorden! I really do like the new pictures. :) 

I do agree with Jannik that a separation of papers extending the tool and underlying theory from those that use the tool is needed now that there are so many papers (and quite a few that are not yet listed but should be).

One other concern I have is the use of external CDN. What's the advantage of that? Yes, it may be slightly faster to load, but I do not think it was an issue before, or was it? Did I just not notice because it is cached for me anyway? Moreover, the downside is that now the repository is not really self-contained anymore it seems to me. Am I too pessimistic here? I am not fundamentally opposed, just want to make sure we do not enter some dependency that comes back to bite us.

**Comment by rsasse** [comment]
Regarding the categorization of papers suggested by Jannik above, I think we can have 3 categories of papers:
- Papers on Tamarin and its Theory
    + includes dissertations, CSF12, CAV13, SP14, CCS15, POST 17
- Tamarin Extensions
    + papers on SAPIC, SLAPIC, human errors, alice&bob notation, etc. go here
- Papers using Tamarin
    + ARPKI, TLS, DNP3: SAv5, 5G, EMV, Selene, etc. go here

For this last category, there are a number of papers using Tamarin that are not yet listed here, but that we should add after all!

Additionally, each category could be split into peer-reviewed (listed first) and non-peer-reviewed (afterwards, this includes all the theses). Inside each I would go newest on top.

Any suggestions or comments on this, maybe from @cascremers ?

**Comment by lordqwerty** [comment]
Why CDN:
* It increases the parallelism available. (Most browsers will only download 3 or 4 files at a time from any given site.)

* It increases the chance that there will be a cache-hit. (As more sites follow this practice, more users already have the file ready.)

* It ensures that the payload will be as small as possible. This makes the time-to-download very small, because it is super compressed and it isn't compressed on the fly.

* It ensures that the user will get a geographically close response.

**Comment by lordqwerty** [comment]
Why CDN:
* It increases the parallelism available. (Most browsers will only download 3 or 4 files at a time from any given site.)

* It increases the chance that there will be a cache-hit. (As more sites follow this practice, more users already have the file ready.)

* It ensures that the payload will be as small as possible. This makes the time-to-download very small, because it is super compressed and it isn't compressed on the fly.

* It ensures that the user will get a geographically close response.

**Comment by rsasse** [comment]
I see, thanks for the explanation Jorden! :+1: 

Another point that we should fix is to host the PDF files as part of this repository, rather than linking to the infsec.ethz.ch DAM for those where this is done. This is not time critical, but we may as well do it now.

**Comment by lordqwerty** [comment]
TAMARIN pixel art monkey now added for favicons. See below for examples.

**Firefox**:
screenshot from 2017-10-17 20-47-50

**Android**:
android

**Comment by cascremers** [comment]
Hi Jorden,

Thanks, great work! If needed, I can make an .ico file with multiple
resolutions (say up to 128x128), which would make the pixel art even
sharper on all platforms. Or I can provide sharp pngs for all resolutions.
Let me know.

Best,

Cas

On Tue, Oct 17, 2017 at 8:52 PM, Jorden Whitefield <notifications@github.com

**Comment by lordqwerty** [comment]
Hi Cas,

If it's easy to produce both then that would be great. 
All we need to do is name them correctly and people internet browsers should do the rest of the work. 

Jorden

**Comment by cascremers** [comment]
Jorden: you've got mail.

**Comment by lordqwerty** [comment]
Thanks Cas! Higher res images added.

**Comment by cascremers** [comment]
Shouldn't the html be updated to include all resolutions and the .ico ?

**Comment by lordqwerty** [comment]
Good catch! 👍

File: index.html [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
 
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
   <title>Tamarin Prover</title>
+  <meta name="description" content="The Tamarin prover is a security protocol verification tool that supports both falsification and unbounded verification in the symbolic model.">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  
+  <link rel="shortcut icon" href="favicon.ico" type="image/png">
```
Comment (cascremers) [inline index.html]: I think the type should be "image/x-icon", and 'rel="icon"' is better ('shortcut' is legacy supported).

**Comment by lordqwerty** [comment]
OK. I think this is all I want to do for this PR. 
I want to create a new branch to:
* Update site version of bootstrap
* Play around with layouts so it works better on mobile
* Add collapse-able nav

**Comment by cascremers** [comment]
Thanks Jorden!

We can do some of the further paper splitting work later, doesn't have to be in this PR.

**Comment by rsasse** [comment]
So now this PR looks like it is done and ready for merging, from my point of view. Is that correct?

The splitting into different categories and adding further papers is indeed an orthogonal issue.

**Comment by lordqwerty** [comment]
All done from my side. I won't be doing any more work on this.

**Pull request merged**